traffic_stats/influxdb_tools/sync/sync_test.go

```diff
@@ -0,0 +1,230 @@
+/*
+Licensed to the Apache Software Foundation (ASF) under one
+or more contributor license agreements.  See the NOTICE file
+distributed with this work for additional information
+regarding copyright ownership.  The ASF licenses this file
+to you under the Apache License, Version 2.0 (the
+"License"); you may not use this file except in compliance
+with the License.  You may obtain a copy of the License at
+
+  http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing,
+software distributed under the License is distributed on an
+"AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+KIND, either express or implied.  See the License for the
+specific language governing permissions and limitations
+under the License.
+*/
+
+package main
+
+import (
+	"encoding/json"
+	"fmt"
+	"testing"
+
+	influx "github.com/influxdata/influxdb/client/v2"
+	"github.com/influxdata/influxdb/models"
+	. "github.com/smartystreets/goconvey/convey"
```

Comment:
I agree with using standard testing. The assertions @sbogacz mentioned are relatively easy to write. If we wanted, we could write a ~100 line library, with e.g. `func AssertEQ(a interface{}, b interface{}, t *testing.T) { if a != b { t.Errorf("expected %v == %v", a, b) } }`. I wouldn't be opposed to such a package, and maintaining that 100-line library is less expensive long-term than making all our tests depend on a third-party library that could stop being maintained at any time.

I'm not universally opposed to dependencies; but again, I don't see the value outweighing the cost here.